tests/test.js

```diff
@@ -60,7 +67,7 @@ test('basic stuff works', function (t) {
     var tests = Array.from(arguments).slice(1)
     tests.forEach(function (test) {
       test.forEach(function (testcase) {
-        t.equal(p.matches(testcase), false, 'should not have matched: ' + testcase + ' ' + p)
+        t.equal(p.matches(testcase), false, 'should not have matched: ' + testcase + ' ' + testcase)
```

Comment:
this change doesnt seem right to me. we're printing that testcase should equal testcase?